### ISSUE TITLE
New version of SOFA

### ISSUE THREAD

**Issue by mhvk** [liberfa/erfa#34]
There is a May 2016 update to SOFA. From the release notes, this seems to include new routines to go from equatorial to ecliptic, so quite possibly relevant. cc @eteq.

**Comment by eteq** [comment]
Oh, interesting - from the astropy perspective, this is *definitely* useful, both for ecliptic and the new long-term precession models (which we probably want to support somehow, but will want to consider carefully how best to do so).

**Comment by phn** [comment]
Hello,

I have updated erfa to use sofa 20160503. The code is at https://github.com/phn/erfa/tree/sofac20160503. 

I followed the steps in RELEASE.rst; upto and including the "make distcheck" step. I think the steps after that are to be followed in the liberfa/erfa repository.

I had to edit the file ``src/Makefile.am`` to include the new files added by SOFA. (See http://www.iausofa.org/current_changes.html). This step is not mentioned in RELEASE.rst and so I have added it.

If this is acceptable, should I raised a new pull request or is there a way to attach my branch to this issue?

Regards,
Prasanth
